### PR TITLE
Add grid fallback, primarily for IE 9, 10 & 11

### LINKED ISSUE #1392: Add grid fallback for article
**Is your feature request related to a problem? Please describe.**
Currently we have implemented the body content layout using CSS grid without a fallback. That means currently on browser like IE 11,  IE 10 [etc](https://caniuse.com/#search=grid) the page appears full width. The page should not be full width on viewports greater than 600px. 

**Describe the solution you'd like**
Add support for devices that dont support grid

**Testing notes**
Manually test browsers/devices to make sure there's no regressing with the implementation of the fallback. It would be good to specifically check iOS7 & IE9 and above as these have also been an issue.

**Additional context**

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by dr3**
Resolves #1392

**Overall change:** Uses max width instead of grid column constraints for IE arising from issues created in https://github.com/bbc/simorgh/pull/1376

**Code changes:**

- Sets max widths on grid items
- Resets max widths on browsers supporting grid

**Implementation**
For groups 4 and 5, this should match to be visually identical. For groups 0-3, as the columns scale, making the grid items match is very tricky, because of this, sections like paragraphs are a rough estimation of their desired width using percentage widths.

image

image

| | |
|-|-|
|image|image|

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: src/app/lib/layoutGrid.js [hunk 1]
```diff
@@ -19,7 +19,31 @@ const group4ColWidth = `6.75rem`;
...
 
 const group5ColWidth = `2.95rem`;
-/* (1280px - (2*16px margins + 19*16px gutters)  / 20 columns = 47.2px = 2.95rem single column width */
+/* (1280px - (2*16px margins + 19*16px gutters) / 20 columns = 47.2px = 2.95rem single column width */
+
+const group4WrapperMaxWidth = `45.5rem`;
+// (6.75rem * 6) + 5*16px gutters = 728 = 45.5 rem
+
+const group5WrapperMaxWidth = `46.4rem`;
+// (2.95rem * 10) + 9*16px gutters = 742.4 = 46.4 rem
```
Comment (dr3) [inline src/app/lib/layoutGrid.js]:
```suggestion
// (2.95rem * 12) + 11*16px gutters = 742.4 = 46.4 rem
```

**Review by pjlee11 — DISMISSED** [review]

File: src/app/lib/layoutGrid.js [hunk 2]
```diff
@@ -40,7 +64,8 @@ export const gelGridMargin = css`
 */
 
 export const layoutGridWrapper = css`
-  display: grid;
+  margin-left: auto;
+  margin-right: auto;
```
Comment (sareh) [inline src/app/lib/layoutGrid.js]: Replacing these two lines with one:
```suggestion
  margin: 0 auto;
```

File: src/app/lib/layoutGrid.js [hunk 3]
```diff
@@ -53,9 +78,18 @@ export const layoutGridWrapper = css`
...
   }
   @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
     grid-template-columns: 1fr repeat(20, minmax(0, ${group5ColWidth})) 1fr;
+    max-width: ${group5WrapperMaxWidth};
+  }
+
+  @supports (display: grid) {
+    display: grid;
+    max-width: initial;
+    margin-left: initial;
```
Comment (sareh) [inline src/app/lib/layoutGrid.js]: Same here with `margin: initial;`

**Review by sareh — CHANGES_REQUESTED** [review]
Looks great when I looked at it on IE9 & IE10. 

We should really think about how best we'd want to test this file, since due to the complexity, it can easily have functionality break without us noticing - particularly when we pull this out into the Psammead component library. 
Can you please open an issue to add unit tests to this layout file? 

I've also added a tiny suggestion for updating margins.

**Comment by dr3** [comment]
Done @sareh :) https://github.com/bbc/simorgh/issues/1396

**Review by sareh — APPROVED** [review]
🎉 Super work!

**Review by pjlee11 — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
👍

**Pull request merged**